circle.yml

```diff
@@ -0,0 +1,12 @@
+dependencies:
+  override:
+    - printf '%s\n' color=false progress=false >~/.npmrc
+    - make setup
+
+machine:
+  node:
+    version: 4
```

Comment:
No good reason. Node 4 came installed on Ubuntu. For a small project it doesn't matter much to me which Node version we use so I'm happy to stick with the default.